src/interfaces.d.ts

```diff
@@ -0,0 +1,86 @@
+export interface VNodeData {
```

Comment:
@staltz Do you know a way to also allow arbitrary keys/values here? When defining your own snabbdom modules you likely need a property not defined here.

```TypeScript
interface VNodeData {
  ... // all properties here
  [ key: string ]: any;
}
```

Is there something like this that we can do?